packages/events/src/FederatedEventMap.ts

```diff
@@ -35,4 +35,8 @@ export type FederatedEventMap = {
 };
 export type FederatedEventEmitterTypes = {
     [K in keyof FederatedEventMap as K | `${K}capture`]: [event: FederatedEventMap[K]];
+} & {
+    'globalpointermove': [event: FederatedPointerEvent];
+    'globaltouchmove': [event: FederatedPointerEvent];
+    'globalmousemove': [event: FederatedPointerEvent];
 };
```

Comment:
```suggestion

export type GlobalFederatedEventMap = {
    globalmousemove: FederatedPointerEvent;
    globalpointermove: FederatedPointerEvent;
    globaltouchmove: FederatedPointerEvent;
};

export type FederatedEventEmitterTypes = {
    [K in keyof FederatedEventMap as K | `${K}capture`]: [event: FederatedEventMap[K]];
} & {
    [K in keyof GlobalFederatedEventMap]: [event: GlobalFederatedEventMap[K]];
};
```

---

Also I wondered if we need to add `GlobalFederatedEventMap` to `IFederatedDisplayObject`:

<https://github.com/pixijs/pixijs/blob/d112742d84da4c1fd61127af3b69519b64a058df/packages/events/src/FederatedEventTarget.ts#L159-L166>

Reply:
I just tested this and it seems that this type never worked in the first place?

I don't seem to get any type information other than the general dom addEventListener typings

Reply:
ok should be all fixed

thanks for tracking this one down for me!